.gitignore

```diff
@@ -41,3 +41,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+*.yml
+config/database.yml
```

Comment:
Why are there two same entries?

Reply:
My bad, it might have been automatically added by own configurations. Oversight on my part, will be careful in the future